src/containers/common/ClassroomCreateFormContainer.jsx

```diff
@@ -1,17 +1,73 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Actions } from 'jumpstate';
 import ClassroomCreateForm from '../../components/common/ClassroomCreateForm';
+import { config } from '../../lib/config';
 
 class ClassroomCreateFormContainer extends React.Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      fields: {
+        name: '',
+        subject: '',
+        school: '',
+        description: ''
+      }
+    };
+
+    this.onSubmit = this.onSubmit.bind(this);
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onChange(event) {
+    const fields = { ...this.state.fields, [event.target.id]: event.target.value };
+    this.setState({ fields });
+  }
+
+  onSubmit(event) {
+    event.preventDefault();
+    // TODO: Add project id(s) associated to classroom create
+    Actions.createClassroom(this.state.fields)
+      .then(() => {
+        Actions.classrooms.setCreateFormVisibility();
+        if (this.props.projectCollection === config.astroProjects) {
```

Comment:
Yeah, I probably should have just commented this whole clause since it's non-functional at the moment. But the basic idea is that I'll want to do a comparison of the app state to the config and if the app state is set to the I2A projects, then auto create the relevant assignments on classroom create.